.github/workflows/build-test-deploy.yml

```diff
@@ -276,11 +277,11 @@ jobs:
       - name: Delete setup job
         run: kubectl delete job -n hubble-commander contracts-setup-job --ignore-not-found=true
       - name: Delete Hubble instances
-        run: helm delete -n hubble-commander hubble-commander
+        run: helm delete -n hubble-commander hubble-commander || true
```

Comment:
Useful when we have no deployments yet. For example at first time.

Reply:
Nice one